frontend/packages/dev-console/src/components/pipelines/modals/start-pipeline/StartPipelineModal.tsx

```diff
@@ -23,7 +23,7 @@ const StartPipelineModal: React.FC<StartPipelineModalProps & ModalComponentProps
   close,
   onSubmit,
 }) => {
-  const userStartedLabel = useUserLabelForManualStart();
+  const userStartedLabel = useUserAnnotationForManualStart();
```

Comment:
`userStartedLabel` gets passed to `submitStartPipeline`'s second parameter, which considers it to be labels.

Reply:
Ah, missed that during refactors. Odd... Thanks!